firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -115,23 +114,34 @@ public static FirebaseInstallations getInstance(@NonNull FirebaseApp app) {
   @NonNull
   @Override
   public Task<String> getId() {
+    return getId(null);
+  }
+
+  private Task<String> getId(AwaitListener awaitListener) {
```

Comment:
add a comment or javadoc for this method, especially mention when the listener will be notified.

Reply:
Done. Thanks.